playbooks/roles/common-server-init/tasks/networking.yml

```diff
@@ -1,13 +1,15 @@
 ---
 
-- name: Add network interface for floating IP
-  when: COMMON_FLOATING_INTERFACE_NAME and COMMON_FLOATING_IP
-  block:
+- name: Add network interface for floating IP (Xenial, Bionic)
+  when: COMMON_FLOATING_INTERFACE_NAME and COMMON_FLOATING_IP and ansible_distribution_release in ["xenial", "bionic"]
 
+  block:
     - name: Add floating IP as network interface alias
       template:
         src: network/interfaces.d/floating-ip.cfg
         dest: /etc/network/interfaces.d/floating-ip.cfg
+        src: netplan/floating-ip.yaml
+        dest: /etc/netplan/floating-ip.yaml
```

Comment:
@giovannicimolin, why is this needed on Xenial?

Reply:
@lgp171188 Oops. This is a typo, fixed!